tester_app/client/app/bundles/HelloWorld/store/helloWorldStore.jsx

```diff
@@ -0,0 +1,24 @@
+import { compose, createStore, applyMiddleware, combineReducers } from 'redux';
+import thunkMiddleware from 'redux-thunk';
+import loggerMiddleware from 'lib/middlewares/loggerMiddleware';
+import reducers from '../reducers';
+import { initalStates } from '../reducers';
+
+export default props => {
```

Comment:
@justin808 @alexfedoseev sorry that was my doing, not Alex's, he had it the right way